docs/Rules/MA0102.md

```diff
@@ -2,7 +2,7 @@
 
 A struct member should use the `readonly` modifier when it does not modify state.
 
-note: This rule is enabled when using C# 8 or later
+> Note: This rule is enabled when using C# 8 or later
```

Comment:
Can you try the syntax explain in https://github.com/community/community/discussions/16925 to format note section?

Reply:
Seems to work. See [here](https://github.com/louis-z/Meziantou.Analyzer/blob/improve-doc/docs/Rules/MA0102.md).